src/sentry/static/sentry/app/components/charts/utils.jsx

```diff
@@ -57,7 +58,12 @@ export function getInterval(datetimeObj, highFidelity = false) {
 
 export function getDiffInMinutes(datetimeObj) {
   const {period, start, end} = datetimeObj;
-  return typeof period === 'string'
-    ? parsePeriodToHours(period) * 60
-    : moment(end).diff(start, 'minutes');
+
+  if (start && end) {
```

Comment:
This is different than the logic before which would prioritize the period over the start & end. If start, end and period are all set should start & end be used first?

Reply:
start/end/period being all set should not be allowed behavior, but I think it's fine to prioritize start and end since it's more specific. We also prioritize it elsewhere.